src/js/services/apiservice.js

```diff
@@ -0,0 +1,39 @@
+export const RegisterUser = (username, surname, passsword, email, phoneNumber, membershipType)=>{
+
+    fetch('http://ereader.retrotest.co.za/api/Db', {
+    method: 'POST',
+    headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+        "usersId": 0,
```

Comment:
Do all your users have the same usersId?

Reply:
Was dummy data that was replaced but it has been removed now, thank you.